### ISSUE TITLE
Unable to zoom out all the way to space

### ISSUE THREAD

**Issue by GabrielBakker** [wrld3d/unity-api#17]
When I play the demos that are included in the WRLD API, i was able to zoom out all the way to space, to the point where the whole map was just a little ball on the screen.

When I attempt to do the same in my game, the game crashes and the API returns the error "Interest point altitude is too high."

It seems like it happens around 20,000 meters or so right now, which is pretty high but still not high enough to see the world as a little ball on the screen.

How are you able to achieve that effect in your demo?

EDIT: I see this is a duplicate question from one somebody asked a couple days ago, so I understand if you want to delete this one. Just know that a lot of us are waiting on this functionality :) Thank you for all your help today figuring this stuff out.

**Comment by Pirh** [comment]
We have managed to reproduce the issue and we have tracked it internally as a bug.

Thanks for reporting this and we will address this in a future release. If we discover any workarounds before then, we will add them here.

(Duplicate of #14)
